abr_control/arms/base_config.py

```diff
@@ -126,6 +140,20 @@ def __init__(self, N_JOINTS, N_LINKS, ROBOT_NAME="robot",
 
         self.gravity = sp.Matrix([[0, 0, -9.81, 0, 0, 0]]).T
 
+    def init_all(self):
```

Comment:
this is useful to get everything generated before entering the control loop, more details in the docstring above

Reply:
hmm, a confusing name though because it doesn't actually initialize everything

it's more like you want to initialize the parts that are being used by the specific controller you're using only ... hm i'm not sure there's a better way to do that then just running `ctrlr.generate()` once before entering the main loop. maybe we could just add a note in the abr_jaco2 repo about that or some such